_aggregations/metric/matrix-stats.md

```diff
@@ -27,60 +43,262 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
 ```
 {% include copy-curl.html %}
 
-#### Example response
+The response containes the aggregated results:
 
 ```json
-...
-"aggregations" : {
-  "matrix_stats_taxful_total_price" : {
-    "doc_count" : 4675,
-    "fields" : [
-      {
-        "name" : "products.base_price",
-        "count" : 4675,
-        "mean" : 34.994239430147196,
-        "variance" : 360.5035285833703,
-        "skewness" : 5.530161335032702,
-        "kurtosis" : 131.16306324042148,
-        "covariance" : {
-          "products.base_price" : 360.5035285833703,
-          "taxful_total_price" : 846.6489362233166
+{
+  "took": 250,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 4675,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "matrix_stats_taxful_total_price": {
+      "doc_count": 4675,
+      "fields": [
+        {
+          "name": "products.base_price",
+          "count": 4675,
+          "mean": 34.99423943014724,
+          "variance": 360.5035285833702,
+          "skewness": 5.530161335032689,
+          "kurtosis": 131.1630632404217,
+          "covariance": {
+            "products.base_price": 360.5035285833702,
+            "taxful_total_price": 846.6489362233169
+          },
+          "correlation": {
+            "products.base_price": 1,
+            "taxful_total_price": 0.8444765264325269
+          }
         },
-        "correlation" : {
-          "products.base_price" : 1.0,
-          "taxful_total_price" : 0.8444765264325268
+        {
+          "name": "taxful_total_price",
+          "count": 4675,
+          "mean": 75.05542864304839,
+          "variance": 2788.1879749835425,
+          "skewness": 15.812149139923994,
+          "kurtosis": 619.1235507385886,
+          "covariance": {
+            "products.base_price": 846.6489362233169,
+            "taxful_total_price": 2788.1879749835425
+          },
+          "correlation": {
+            "products.base_price": 0.8444765264325269,
+            "taxful_total_price": 1
+          }
         }
-      },
-      {
-        "name" : "taxful_total_price",
-        "count" : 4675,
-        "mean" : 75.05542864304839,
-        "variance" : 2788.1879749835402,
-        "skewness" : 15.812149139924037,
-        "kurtosis" : 619.1235507385902,
-        "covariance" : {
-          "products.base_price" : 846.6489362233166,
-          "taxful_total_price" : 2788.1879749835402
+      ]
+    }
+  }
+}
+```
+
+The following table describes the response fields.
+
+| Statistic    | Description |
+| :---         | :---        |
+| `count`      | The number of documents sampled for the aggregation. |
+| `mean`       | The average value of the field computed from the sample. |
+| `variance`   | The square of deviation from the mean, a measure of data spread. |
```

Comment:
Line 115: Is "about" the right word here?

Reply:
It's understandable but I'll change to "relative to" to be more precise.